packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2695,6 +2709,25 @@ class _InspectorOverlayLayer extends Layer {
   }) {
     return false;
   }
+
+  /// Return whether or not a render object belongs to this inspector widget
+  /// tree.
+  /// The inspector selection is static, so if there are multiple inspector
+  /// overlays in the same app (i.e. an storyboard), a selected or candidate
+  /// render object may not belong to this tree.
+  bool _isInInspectorRenderObjectTree(RenderObject child) {
+    RenderObject inspectorRoot;
```

Comment:
inspectorRoot seems like it is unused. remove.

Reply:
Good catch. It was a remain of a different implementation I had for that.